BILLthebuilder/automart
# #166889085 Improve test coverage

#### What does this PR do?
* It improves test coverage
#### Description of Task to be completed?
* Adding more test cases and refactoring code
* Removing failing tests on travis ci
#### How should this be manually tested?
* Clone the repository
* Checkout to the `bg-fix-travis-166813835` branch
* `npm install`
* `npm test`
* All tests should pass with increased coverage by 10%
